Include/internal/pycore_code.h

```diff
@@ -24,9 +27,117 @@ struct _PyOpcache {
     char optimized;
 };
 
+
+// XXX Can we do this with an enum instead?
+typedef unsigned char _PyFastLocalKind;
+#define CO_FAST_POSONLY     0x01
+#define CO_FAST_POSORKW     0x02
+#define CO_FAST_VARARGS     0x04
+#define CO_FAST_KWONLY      0x09
+#define CO_FAST_VARKWARGS   0x10
+#define CO_FAST_LOCALONLY   0x20
+#define CO_FAST_CELL        0x40
+#define CO_FAST_FREE        0x80
+#define CO_FAST_LOCAL (CO_FAST_POSONLY | CO_FAST_POSORKW | CO_FAST_VARARGS | \
+                       CO_FAST_KWONLY | CO_FAST_VARKWARGS |                  \
+                       CO_FAST_LOCALONLY)
+#define CO_FAST_ANY (CO_FAST_LOCAL | CO_FAST_CELL | CO_FAST_FREE)
+
+
+struct _PyCodeConstructor {
```

Comment:
This name feels kind of awkward -- a "constructor" in common parlance is a function or method of some sort, not a data structure. Maybe `_PyCodeConstructionArgs` or `_PyCodeNewArgs`?

Reply:
I borrowed from the name we use for the same purpose with `_PyFrame_New_NoTrack()` -- I agree that the name is awkward but was trying to stick with precedent. 🙂  I don't mind calling it something different.

Reply:
Hm, okay, the precedent is fine then.